renku/cli/_ascii.py

```diff
@@ -88,15 +94,21 @@ def __iter__(self):
 
     def node_text(self, node):
         """Return text for a given node."""
-        result = [
-            _format_sha1(self.graph, node) + ' ' +
-            self.graph._format_path(node[1])
-        ]
-        workflow = self.graph.G.nodes[node].get('workflow_path')
-        if workflow:
+        formatted_sha1 = _format_sha1(self.graph, node)
+        result = [formatted_sha1 + ' ' + self.graph._format_path(node[1])]
+        workflow_path = self.graph.G.nodes[node].get('workflow_path')
+        if workflow_path:
+            workflow_path = click.style(
+                self.graph._format_path(
+                    os.path.normpath(os.path.join(node[1], workflow_path))
+                ),
+                fg='blue'
```

Comment:
Would be helpful to use a constant that describes the semantics of the color instead of specifying the color directly.

Reply:
I will leave it for now and create an issue.